python/ray/train/constants.py

```diff
@@ -61,7 +61,7 @@ def _get_ray_train_session_dir() -> str:
 
 
 def _v2_migration_warnings_enabled() -> bool:
-    return env_bool(ENABLE_V2_MIGRATION_WARNINGS_ENV_VAR, False)
+    return env_bool(ENABLE_V2_MIGRATION_WARNINGS_ENV_VAR, True)
```

Comment:
btw should we also include in the messages that this warning can be turned off by setting this env var?